package.json

```diff
@@ -4,15 +4,19 @@
   "private": true,
   "main": "expo-router/entry",
   "scripts": {
-    "start": "expo start",
+    "disable-storybook": "cross-env STORYBOOK_ENABLED='false'",
+    "enable-storybook": "cross-env STORYBOOK_ENABLED='true'",
```

Comment:
That looks weird to me, why would we want to disable storybook ?
I think we always want to have STORYBOOK_ENABLE=true on development (so to prefix concerned scripts by STORYBOOK_ENABLED=true)

Reply:
I'm not sure about this, the problem is that you can't run Storybook and the app simultaneously on the same port. Only if we are executing `yarn storybook` on port 8081 will launch only Storybook, while on an other terminal execute `yarn dev` on port 8082 will run just the app. but on two diffrent emulators The purpose of `cross-env` is to specify which environment to use, allowing us to choose whether to start Storybook or the app.

Reply:
Note: we can do it in the same app (diffrent route) but the problem is on react native we have to reload many times the app to retest and it will be so innoying if click open menu show storybook on each test reload